tests/pkg_fff/Makefile

```
@@ -0,0 +1,6 @@
+include ../Makefile.tests_common
+USEPKG += fff
+# sltb001a always enables periph_i2c.
+# This clashes with the mock of periph_i2c.
+BOARD_BLACKLIST := sltb001a
```

```suggestion
FEATURES_BLACKLIST := periph_i2c
```
@maribu @aabadie so you would suggest to use this?

---

Thanks @maribu for confirming the initial idea is ok :)

Better use:

```
FEATURES_BLACKLIST += periph_i2c
```

And add a comment explaining that this application cannot be used if periph_i2c is pulled in because of conflict.

---

Should `make info-boards-supported` still contain the `sltb001a` board after this change?

---

It shouldn't once rebase on https://github.com/RIOT-OS/RIOT/pull/17100

---

Changed this, rebased and squashed again.